source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+    **Note:** Only updating the mobile apps, or updating the mobile apps before the Mattermost server, can result in incompatibility.
+
+----
+
+####################################
+Appendix A: Troubleshooting and FAQs
+####################################
+
+We highly recommend you checkout our `Mobile FAQ <https://docs.mattermost.com/mobile/mobile-faq.html>`_ and `Mobile Troubleshooting documentation <https://docs.mattermost.com/mobile/mobile-troubleshoot.html>`_. The most common questions we received have been answered there. However, it’s important to call out a few common items customers run into.
+
+*******************************
+Data Security on Mobile Devices
+*******************************
+
+- `How is data handled on a device after an account is deleted? <https://docs.mattermost.com/mobile/mobile-faq.html#how-is-data-handled-on-mobile-devices-after-a-user-account-is-deactivated>`_
+- `What post metadata is sent in mobile push notifications? <https://docs.mattermost.com/mobile/mobile-faq.html#what-post-metadata-is-sent-in-mobile-push-notifications>`_
+- `What are my options for securing the mobile apps? <https://docs.mattermost.com/mobile/mobile-faq.html#what-are-my-options-for-securing-the-mobile-apps>`_
+- `What are my options for securing push notifications? <https://docs.mattermost.com/mobile/mobile-faq.html#what-are-my-options-for-securing-push-notifications>`_
+
+***********************
+Corporate Proxy Servers
+***********************
+
+`How do I receive mobile push notification if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`_
+
+- `Deploy Mattermost in a proxy-aware configuration with a pre-proxy relay <https://docs.mattermost.com/mobile/mobile-faq.html#deploy-mattermost-in-a-proxy-aware-configuration-with-a-pre-proxy-relay>`_
+- `Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment <https://docs.mattermost.com/mobile/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment>`_
+- `Whitelist Mattermost push notification proxy to bypass your corporate proxy server <https://docs.mattermost.com/mobile/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server>`_
+- `Run App Store versions of the Mattermost mobile apps <https://docs.mattermost.com/mobile/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps>`_
```

Comment:
App Store only talks about Apple

Reply:
Again “App Store” is the generic term used within the docs. “You can build the app from source and distribute it within your team or company either using the App Stores, Enterprise App Stores or EMM providers, or another way of your choosing.” https://developers.mattermost.com/contribute/mobile/build-your-own/

Reply:
resolved in meeting